### DISCUSSION TITLE
have braces on same line ?

### DISCUSSION THREAD

**Discussion by ArtooS** [PHP-CS-Fixer/PHP-CS-Fixer#8071] [Q&A]
shouldn't putting the following in the   "php-cs-fixer.rules":  section of the settings.json file show the curly braces on the same line 
 
    "braces" => [
      "allow_single_line_closue" => true,
      "control_structures_opening_brace" => 'same_line',
      "position_after_functions_and_oop_constructs" => 'same',
      "control_structures_opening_brace" => 'same_line',
      "classes_opening_brace" => 'same_line',
      "functions_opening_brace" => 'same_line'
    ]

For some reaon, when I format a php file with saving the curly braces appear on the next line.

**Comment by Wirone** [comment]
Hey!

We identified that `braces` fixer is too complex and has too many responsibilities. With that, [we decided to deprecate it](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/4885).

That also means that we will not be extending, changing the behaviour or fixing anything around `braces` fixer. Please, use [replacement rules](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/basic/braces.rst#this-rule-is-deprecated-and-will-be-removed-on-next-major-version) instead.

Also, it seems like you're a Pint user (`settings.json`), so maybe you should ask Pint's maintainers ☺️.
